serverless/installing-service-mesh-control-plane.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// serverless/installing-openshift-serverless.adoc
+
+= Installing the ServiceMeshControlPlane
+
+Service Mesh is comprised of a data plane and a control plane. After you install the ServiceMesh operator, you can install the control plane. The control plane manages and configures the sidecar proxies to enforce policies and collect telemetry. The following procedure installs a version of the ServiceMesh control plane that acts as an ingress to your applications.
+
+====
+[NOTE] 
+You need to install the control plane into the `istio-system` namespace. Other namespaces are currently not supported.
+====
+
+
+[source,yaml]
+.Sample servicemesh-installation.yaml
+----
+apiVersion: maistra.io/v1
+kind: ServiceMeshControlPlane
+metadata:
+ name: basic-install
+ namespace: istio-system <1>
+spec:
+ istio:
+   global:
+     multitenant: true
+     proxy:
+       autoInject: disabled
+     omitSidecarInjectorConfigMap: true
+     disablePolicyChecks: false
+   istio_cni:
+     enabled: true
+   gateways:
+     istio-ingressgateway:
+       autoscaleEnabled: false
+       type: LoadBalancer
+     istio-egressgateway:
+       enabled: false
+     cluster-local-gateway:
+       autoscaleEnabled: false <1>
+       enabled: true
+       labels:
+         app: cluster-local-gateway
+         istio: cluster-local-gateway
+       ports:
+         - name: status-port
+           port: 15020
+         - name: http2
+           port: 80
+           targetPort: 80
+         - name: https
+           port: 443
+   mixer:
+     enabled: false
+     policy:
+       enabled: false
+     telemetry:
+       enabled: falsely 
+   pilot:
+     autoscaleEnabled: false
+     sidecar: false
+   kiali:
+     enabled: false
+   tracing:
+     enabled: false
+   prometheus:
+     enabled: false
+   grafana:
+     enabled: false
+   sidecarInjectorWebhook:
+     enabled: false
+----
+<1> Autoscaling is disabled in this version. This release is not intended for production use.
+
+
+====
+[NOTE]
+Running ServiceMesh with a sidecar injection enabled with Openshift Serverless is currently not recommended. 
+====
+
+
+.Prerequisite
+* An account with cluster administrator access.
+* The ServiceMesh operator must be installed. 
+
+
+.Procedure
+. Copy the sample YAML file into a `smcp.yaml` file.
+. Apply the YAML file using the command:
++
+----
+ oc apply -f smcp.yaml
+----
+
+
+// Verifying the control plane installation
+include:modules/ossm-control-plane-verify.adoc[leveloffset=+1]
```

Comment:
This module has this spot in it:

```
.Procedure

[NOTE]
====
The name of the resource is `istio-installation`.
====
```

This renders funny:

![image](https://user-images.githubusercontent.com/3442316/63619715-6ece1080-c5bd-11e9-84fb-94d67ebe458e.png)

Suggest moving the note admonition either before the `.Procedure` or elsewhere in a step.